lib/chef-dk/command/verify.rb

```diff
@@ -452,6 +451,71 @@ def foo
         end
       end
 
+      unless Chef::Platform.windows?
+        add_component "delivery-cli" do |c|
+          # We'll want to come back and revisit getting unit tests added -
+          # currently running the tests depends on cargo , which is not included
+          # in our package.
+          c.base_dir = "bin"
+          c.smoke_test do
+            tmpdir do |cwd|
+              sh!("delivery setup --user=shipit --server=delivery.shipit.io --ent=chef --org=squirrels", cwd: cwd)
+            end
+          end
+        end
+
+        add_component "git" do |c|
+          c.base_dir = "embedded/bin"
+          c.smoke_test do
+            sh!("#{bin("git")} config -l")
+          end
+          c.integration_test do
```

Comment:
While I agree, I'm going to make a different card for that. This PR has already lasted too long and I don't want to scope creep into getting integration tests also working.